selfdrive/car/hyundai/values.py

```diff
@@ -59,7 +59,7 @@ class CAR:
   KIA_FORTE = "KIA FORTE E 2018 & GT 2021"
   KIA_K5_2021 = "KIA K5 2021"
   KIA_NIRO_EV = "KIA NIRO EV 2020"
-  KIA_NIRO_HEV = "KIA NIRO HYBRID 2019"
+  KIA_NIRO_PHEV = "KIA NIRO HYBRID 2019"
```

Comment:
From this PR, this car is actually a plug-in hybrid: https://github.com/commaai/openpilot/pull/21481